lmdeploy/lite/quantization/awq.py

```diff
@@ -57,6 +61,10 @@
     'DecoderLayer': {
         'self_attn.W_pack': ['self_attn.o_proj'],
         'mlp.up_proj': ['mlp.down_proj']
+    },
+    'GLMBlock': {
```

Comment:
Can we remove the `GLMBlock` item?

Reply:
No, we can't do that. But we may remove the commented lines below it.